src/pages/authentication/overview.md

```diff
@@ -35,9 +35,9 @@ Scopes define the permissions requested by an app for granting during authentica
 Apps may request any of the following scopes:
 
 | Scope          | Definition                                                                           |
-| -------------- | ------------------------------------------------------------------------------------ |
+| -------------- | ------------------------------------------------------------------------------------ | --- |
```

Comment:
this creates a third empty column, can we remove that?

Reply:
If you rebase from master this will be fixed

Reply:
Agreed, I'll remove via rebase